frontend/src/widgets/TotalHrsAndGranteeGraph.js

```diff
@@ -0,0 +1,226 @@
+import React, { useState, useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import Plotly from 'plotly.js-basic-dist';
+import { Grid } from '@trussworks/react-uswds';
+import withWidgetData from './withWidgetData';
+import DateTime from '../components/DateTime';
+import './TotalHrsAndGranteeGraph.css';
+
+export function Tooltip(props) {
+  const {
+    show, x, y, text,
+  } = props;
+  return show ? <span className="ttahub--total-hrs-grantee-graph-tooltip" style={{ left: x, top: y - 50 }}>{text}</span> : null;
+}
+
+Tooltip.propTypes = {
+  show: PropTypes.bool.isRequired,
+  x: PropTypes.number.isRequired,
+  y: PropTypes.number.isRequired,
+  text: PropTypes.number.isRequired,
+};
+
```

Comment:
Might be worthwhile to cleanup the eslint-disables that we don't need and the commented out stuff. It also might be nice to have a comment that explains which object in the data array is which (i.e. data[0] is Technical Assistance or w/e... or you could destructure like const [technicalAssistance, training, both, etc] = data... might be a little more readable in the future? Just a suggestion

Reply:
I have cleaned up the code based on these suggestions.

Reply:
One other suggestion, and again, not a deal breaker, would be moving that tooltip component up into it's own file since we're using identical code two places